main.tf

```diff
@@ -31,4 +36,6 @@ module "services" {
   cloudtrail_sns_topic_arn = module.cloudtrail_organizational.sns_topic_arn
   services_assume_role_arn = aws_iam_role.cloudvision_role.arn
   tags                     = var.tags
+
+  cloudbench_account_ids = [var.aws_organization_cloudvision_account_id] // TODO populate this with all accounts in org
```

Comment:
Need to sync on how to determine list of accountIDs. Is it possible to fetch the list of accounts within an orginization? Will we have permissions to create roles/policies in these child accounts?

Reply:
Perfect! I'll use the same approach to create the cloud-bench role in the same member account, and worry about multiple accounts in a separate PR